db/migrate/20230712140355_add_fields_to_likes.rb

```diff
@@ -0,0 +1,6 @@
+class AddFieldsToLikes < ActiveRecord::Migration[7.0]
+  def change
+    # add_column :likes, :author_id, :integer
+    # add_column :likes, :post_id, :integer
+  end
+end
```

Comment:
- [ ] I observed you newly added commented-out codes after the last review, kindly ensure that the migration file is not empty for smooth running of the database. 👍